components/AboutPage.tsx

```diff
@@ -0,0 +1,85 @@
+import Image from 'next/image';
+import React from 'react';
+import PageWrapper from '../components/PageWrapper';
+import { Lang } from '../lib/types';
+
+interface Props {
+  language: Lang;
+}
+
+const AboutPage: React.FC<Props> = ({ language }) => {
+  const content = {
```

Comment:
this is just a stylistic thing, and please disregard if you prefer it this way. but i would move this variable _out_ of the component, and put it at the _bottom_ of the file. my reasoning is this: my guess is that 9/10 times you'll be working on this file, you'll be thinking about the component, styling, etc., and not the text. it's nice not to have to wade through all these long strings to get to the part that you're likely to be using the most.  so, if it were me, i would move this to the bottom of the file (and the same with the other static page components).

but again, if this feels nicer to you, by all means keep it here. totally a preference thing, one way is not better than another

Reply:
some of the components use props in these strings, so you couldn't move those out (as easily), so maybe ignore those, if you do decide to move them. (or don't move any of them)